include/abb_librws/rws_interface.h

```diff
@@ -63,6 +63,16 @@ class RWSInterface
      * \brief The system's name.
      */
     std::string system_name;
+
+    /**
+     * \brief The system's type.
+     */
+    std::string system_type;
+
+    /**
+     * \brief The system's active options.
+     */
+    std::vector<std::string> system_options;
```

Comment:
Perhaps call it `active_system_options` then?

Reply:
Clarified in 90ab7a9.

I just removed the `active` part in the comment. The options are already implicitly present in the system if they show up in the `system_options` list.